packages/tools/devtools/devtools-core/src/test/DefaultVisualizers.test.ts

```diff
@@ -494,72 +486,112 @@ describe("DefaultVisualizers unit tests", () => {
 												"0": {
 													children: {
 														type: {
-															value: "DefaultVisualizer_SharedTree_Test.child-item",
+															value: 'DefaultVisualizer_SharedTree_Test.List<["DefaultVisualizer_SharedTree_Test.child-item"]>',
 															typeMetadata: "string",
 															nodeKind: VisualNodeKind.ValueNode,
 														},
 														fields: {
 															children: {
-																childField: {
-																	children: {
-																		"0": {
-																			children: {
-																				type: {
-																					value: "com.fluidframework.leaf.string",
-																					typeMetadata:
-																						"string",
-																					nodeKind:
-																						VisualNodeKind.ValueNode,
-																				},
-																				value: {
-																					value: "Hello world!",
-																					typeMetadata:
-																						"string",
-																					nodeKind:
-																						VisualNodeKind.ValueNode,
-																				},
-																			},
-																			nodeKind:
-																				VisualNodeKind.TreeNode,
-																			typeMetadata: "object",
-																		},
-																	},
-																	nodeKind:
-																		VisualNodeKind.TreeNode,
-																	typeMetadata: "object",
-																},
-																childData: {
+																"": {
```

Comment:
@jikim-msft @Josmithr I think your input here would be valuable. I don't quite get if the changes to this object should be expected or not.